runtime/lib/backend/lightning/lightning_dynamic/LightningSimulator.hpp

```diff
@@ -85,6 +85,27 @@ class LightningSimulator final : public Catalyst::Runtime::QuantumDevice {
         return res;
     }
 
+    inline auto simulateDraw(const std::vector<double> &probs, int8_t postselect) const -> bool
```

Comment:
what would you think of adding this method to `common/Utils.hpp` now this is used in multiple translation units?

Reply:
I think you are right. Done.